### PR TITLE
Add 4.61.0 changelog (ga)

### PR DESCRIPTION
**Pull request by rileykarson**
- [x] Merge https://github.com/GoogleCloudPlatform/magic-modules/pull/7665
- [x] Update to include https://github.com/GoogleCloudPlatform/magic-modules/pull/7665
- [x] Merge https://github.com/GoogleCloudPlatform/magic-modules/pull/7666
- [x] Update to include https://github.com/GoogleCloudPlatform/magic-modules/pull/7666

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
 ## 4.61.0 (Unreleased)
 
+BREAKING CHANGES:
+* cloudrunv2: set a default value of 3 for `max_retries` in `google_cloud_run_v2_job`. This should match the API's existing default, but may show a diff at plan time in limited circumstances as drift is now detected ([#14223](https://github.com/hashicorp/terraform-provider-google/pull/14223))
+
+FEATURES:
+* **New Data Source:** `firebase_android_app_config` ([#14202](https://github.com/hashicorp/terraform-provider-google/pull/14202))
```
Comment (c2thorn) [inline CHANGELOG.md:7]:
```suggestion
* **New Data Source:** `google_firebase_android_app_config` ([#14202](https://github.com/hashicorp/terraform-provider-google/pull/14202))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+* compute: added support for `stack_type` to `google_compute_ha_vpn_gateway` ([#14141](https://github.com/hashicorp/terraform-provider-google/pull/14141))
+* container: added support for `ephemeral_storage_local_ssd_config` to `google_container_cluster.node_config`, `google_container_cluster.node_pools.node_config`, `google_container_node_pool.node_config` ([#14150](https://github.com/hashicorp/terraform-provider-google/pull/14150))
+* dlp: Changed `dictionary`, `regex`, `regex.group_indexes` and `large_custom_dictionary` fields in `google_data_loss_prevention_stored_info_type` to be update-in-place ([#14207](https://github.com/hashicorp/terraform-provider-google/pull/14207))
+* network_services: increased the max count for `route_rule` to 200 on `google_network_services_edge_cache_service` ([#14224](https://github.com/hashicorp/terraform-provider-google/pull/14224))
+* logging: added support for `disabled` to `google_logging_metric` ([#14198](https://github.com/hashicorp/terraform-provider-google/pull/14198))
```
Comment (c2thorn) [inline CHANGELOG.md:23]: looks like we've had both networkservices and network_services in the past, so up to you on the spelling, but the ordering should change


```suggestion
* logging: added support for `disabled` to `google_logging_metric` ([#14198](https://github.com/hashicorp/terraform-provider-google/pull/14198))
* networkservices: increased the max count for `route_rule` to 200 on `google_network_services_edge_cache_service` ([#14224](https://github.com/hashicorp/terraform-provider-google/pull/14224))
```

File: CHANGELOG.md (hunk 1 above)
Comment (rileykarson) [reply CHANGELOG.md:7]: Good catch! I saw one issue and not the other here

File: CHANGELOG.md (hunk 2 above)
Comment (rileykarson) [reply CHANGELOG.md:23]: lol I sounded the alphabet and convinced myself this one was `mnlop` not `lmnop` 🤦

**Review by c2thorn — APPROVED** [review]

**Pull request merged**